types/sinon/index.d.ts

```diff
@@ -775,17 +775,13 @@ declare namespace Sinon {
         (obj: any): SinonMock;
     }
 
-    type SinonTimerId = FakeTimers.TimerId;
```

Comment:
this doesn't exist anymore. right now sinon's types define the return type as `any`, but i imagine ultimately it will instead just be `number|NodeTimer` (whatever the interface of node's timers is called)